apps/source_academy/lib/source_academy/user.ex

```diff
@@ -0,0 +1,48 @@
+defmodule SourceAcademy.User do
+  @moduledoc """
+    The User entity contains basic user data such as name and e-mail.
+    A user can have many authorisations from different providers.
+  """
+  use Ecto.Schema
+  import Ecto.Changeset
+  alias SourceAcademy.Repo
+
+  @type t :: %__MODULE__{}
+
+  schema "users" do
+    field :name, :string
+    field :email, :string
+    field :is_admin, :boolean
+
```

Comment:
Provide a nullable string field to indicate password reset ID, this is the only part of backoffice where student and staff will see.
Then provide a nullable timestamp to indicate whether this ID expire and password recovery must be requested agan.
We implement it later.

Something like backoffice.com/password_reset?id=somerandomlygeneratedid should go to a page where the user can input new password. I wish Guardian has this feature built in